x-pack/plugins/data_enhanced/public/search/search_interceptor.ts

```diff
@@ -51,7 +51,7 @@ export class EnhancedSearchInterceptor extends SearchInterceptor {
     if (this.longRunningToast) return;
     this.longRunningToast = this.deps.toasts.addInfo(
       {
-        title: 'Your query is taking awhile',
+        title: 'Your query is taking a while',
```

Comment:
I remember looking this up when implementing the feature.
I think `awhile` is actually the right form, but I also find it weird.

Reply:
Yeah, it's a bit confusing for me too 🙃 

The way I understand it is `awhile` == `for a while` so it would read something like: "Your query is taking for a while" which can also work 🤷‍♂️